src/redux/categories/categories.jsx

```diff
@@ -1,20 +1,21 @@
 const CHECK_STATUS = 'book-store/categories/CHECK_STATUS';
 // Reducer
-export const categoriesReducer = (state = [], action) => {
+const categoriesReducer = (state = [], action) => {
   switch (action.type) {
     case CHECK_STATUS:
-      return action.text;
+      return action.value;
     default:
       return state;
   }
 };
 
-// Action Creators
-
-export const CheckStatus = () => {
+// Action
+export const checkStatus = () => {
   const status = {
     type: CHECK_STATUS,
-    text: 'Under construction',
+    value: 'Under construction',
   };
   return status;
 };
+
+export default { categoriesReducer };
```

Comment:
- [ ] Right now your categories page does not seem to be working properly as clicking the button is not showing the `Under Construction` text. It might be due to a slight syntactical error in this export function. Kindly resolve this issue and make sure the Categories page is displaying the Under Construction text